packages/host/app/lib/matrix-classes/room.ts

```diff
@@ -67,7 +68,16 @@ export default class Room {
       ?.get('')?.event.content ?? {
       enabledEventIds: [],
       disabledEventIds: [],
-    }) as { enabledEventIds: string[]; disabledEventIds: string[] };
+      enabledCards: [],
+      disabledCards: [],
+      commandDefinitions: [],
+    }) as {
+      enabledEventIds: string[];
+      disabledEventIds: string[];
+      enabledCards: SerializedFile[];
+      disabledCards: SerializedFile[];
```

Comment:
I'm a bit confused why a skill needs disabled cards. Can you explain?

Reply:
Is it better to rename it to enabledSkills and disabledSkills? skill is a card that is also uploaded.

Reply:
or `enabledSkillCards` / `disabledSkillCards`